app/assets/javascripts/central.js

```diff
@@ -7,6 +7,13 @@ var ProjectSearchView = require('views/project_search_view');
 var ProjectVelocityView = require('views/project_velocity_view');
 
 import TourController from 'controllers/tour/TourController';
+import Pusher from 'pusher-js';
+
+var pusherApiKey = process.env.PUSHER_APP_KEY;
+var pusherCluster = process.env.PUSHER_APP_CLUSTER;
+var boardSocket = new Pusher(pusherApiKey,{
```

Comment:
What do you think about extracting the instantiation of Pusher to a separate file? Then you would only need to import the socket and use it. Something like:

```js
import { boardSocket } from './pusherSockets';
```